dlang/plugin-impl/build.gradle.kts

```diff
@@ -1,156 +1,55 @@
-                "--add-opens=java.base/java.util=ALL-UNNAMED",
-                "--add-opens=java.base/sun.nio.ch=ALL-UNNAMED",
-                "--add-opens=java.base/jdk.internal.vm=ALL-UNNAMED",
-                "--add-opens=java.desktop/sun.awt=ALL-UNNAMED",
-                "--add-opens=java.desktop/sun.awt.image=ALL-UNNAMED",
-                "--add-opens=java.desktop/sun.awt.X11=ALL-UNNAMED", // Linux only
-                "--add-opens=java.desktop/sun.font=ALL-UNNAMED",
-                "--add-opens=java.desktop/sun.swing=ALL-UNNAMED",
-                "--add-opens=java.desktop/sun.java2d=ALL-UNNAMED",
-                "--add-opens=java.desktop/java.awt=ALL-UNNAMED",
-                "--add-opens=java.desktop/java.awt.peer=ALL-UNNAMED",
-                "--add-opens=java.desktop/java.awt.event=ALL-UNNAMED",
-                "--add-opens=java.desktop/javax.swing=ALL-UNNAMED",
-                "--add-opens=java.desktop/javax.swing.text.html=ALL-UNNAMED",
-                "--add-opens=java.desktop/javax.swing.plaf.basic=ALL-UNNAMED",
-                "--add-opens=java.desktop/com.sun.java.swing.plaf.gtk=ALL-UNNAMED", // Linux only
-                "--add-opens=java.desktop/com.apple.eawt.event=ALL-UNNAMED",  // Mac OSX only
-            )
-            systemProperty("idea.debug.mode", "true")
-            // The package names have to be prefixed with a #
-            systemProperty("idea.log.debug.categories", "#io.github.intellij.dlanguage")
-            //systemProperty("idea.log.trace.categories"], "#io.github.intellij.dlanguage")
-
-            // Enable internal mode (https://plugins.jetbrains.com/docs/intellij/enabling-internal.html)
-            systemProperty("idea.is.internal", "true")
-
-            // idea.auto.reload.plugins=false
-            systemProperty("ide.show.tips.on.startup.default.value", "false")
-
-            // Set these properties to test alternative languages
-            // systemProperty("user.language"], "de")
-            // systemProperty("user.country"], "DE")
-
-            // jbrVersion = null
-            // jbrVersion = "11_0_4b304.69"
+    }
+    publishing {
+        token = provider {
+            System.getenv("JETBRAINS_TOKEN")
         }
-
-        publishPlugin {
-            enabled = true
-            token.set(provider {
-                System.getenv("JETBRAINS_TOKEN")
-            })
-            channels.set(publishChannels)
+        channels = listOf(properties("publishChannels"))
+    }
+    verifyPlugin {
+        ides {
+            recommended()
         }
     }
 }
 
-
-// take the version number defined in gradle build and use that in plugin.xml
-tasks.register<Copy>("initConfig") {
-    from("src/main/resources") {
-        include("**/plugin.xml")
-        expand("version" to version)
+tasks {
+    runIde {
+        jvmArgumentProviders += CommandLineArgumentProvider {
+            listOf("-Dide.show.tips.on.startup.default.value=false")
```

Comment:
I think there's a property to disable _code with me_ as well, may be worth adding if that's still included when running locally